contrib_bots/lib/sched.py

```diff
@@ -0,0 +1,209 @@
+        time.sleep(wakeup["timeout_in_seconds"]+1)
+        schedule.run_pending()
+        time.sleep(wakeup["timeout_in_seconds"]+7)
+        schedule.every(1).seconds.do(start_up, client)
+
+        return '''
+            Users should preface messages with "@every" or '@cancel' or '@recover'.
+
+            Before running this, make sure to create a stream
+            called "alert" that your API user can send to.
+            '''
+
+    def triage_message(self, message):
+        # return True if we want to (possibly) response to this message
+
+        original_content = message['content']
+
+        # This next line of code is defensive, as we
+        # never want to get into an infinite loop of posting follow
+        # ups for own follow ups.
+        is_link = (original_content.startswith('@every') or
+                   original_content.startswith('@cancel') or
+                   original_content.startswith('@recover'))
+
+        return is_link
+
+    def handle_message(self, message, client, state_handler):
+
+        original_content = message['content']
+        original_sender = message["sender_email"]
+        a = str(randint(1, 1000000))
+        key = dict(
+            type='stream',
+            to='alert',
+            subject="All alerts",
+            content=original_content+" "+a,
+            )
+
+        # List needed to verify if time interval was valid.
+        weekdays = ["mondays", "tuesdays", "wednesdays", "thursdays",
+                    "fridays", "saturdays", "sundays", "days"]
+
+        weekday = ["monday", "tuesday", "wednesday", "thursday",
+                   "friday", "saturday", "sunday", "day"]
+
+        intervals = ["hour", "hours", "minute", "minutes"]
+        # Verify that @every command meets the syntax required for data indexing.
+        if len(original_content.split()) > 3:
+            if (original_content.startswith('@every') and original_content.split()[1].isdigit() and
+                (original_content.split()[2] in weekdays or original_content.split()[2] in weekday)
+               and is_time_format(original_content.split()[4])):
+                with open('events.json') as data_file:
+                    data = json.load(data_file)
+                if key not in data:
+                    data.append(key)
+                with open('events.json', 'w') as data_file:
+                    json.dump(data, data_file)
+                coefficient = int(original_content.split()[1])
+                interval = original_content.split()[2]
+                timer = original_content.split()[4]
+                if interval in weekdays and coefficient < 2:
+                    interval = interval[:-1]
+                if coefficient > 1 and interval in weekday:
+                    interval = interval+"s"
+                task = schedule.every(coefficient)
+                task = getattr(task, interval)
```

Comment:
`task.interval` should do the same thing. Also, you could store the result in some `task_interval` variable for better readability.